File: android/app/src/main/java/clientleger/faismoiundessin/repository/data/GameInfo.kt
package clientleger.faismoiundessin.repository.data

import kotlinx.serialization.Serializable

@Serializable
data class GameInfo(
    var type: ArrayList<String>,
    var difficulty: ArrayList<String>,
    var humanPlayers: ArrayList<String>,
    var virtualPlayers: ArrayList<String>,
    var gameID: ArrayList<String>,
    var chatID: ArrayList<String>
)

File: client/src/app/classes/game-info.ts
export interface GameInfo {
    gameType: string;
    Difficulty: string;
    HumanPlayers: number;
    VirtualPlayers: number;
    GameID: number;
    ChatID: number;
    State: string;
    WordToGuess: string;
    Teams: TeamGame[];
    Drawer: string;
}

export interface TeamGame {
    Players: string[];
    IsCurrentlyPlayin: boolean;
    Score: number;
}
